packages/websocket-plugin/src/websocket-handler.ts

```diff
@@ -7,7 +7,8 @@ import {
   SendWebSocketMessage,
   NGXS_WEBSOCKET_OPTIONS,
   NgxsWebsocketPluginOptions,
-  WebsocketMessageError
+  WebsocketMessageError,
```

Comment:
add WebsocketMessageError, WebSocketDisconnected in public_api

Reply:
@splincode where? In public_api.ts?? Why?

Reply:
https://github.com/ngxs/store/blob/master/packages/websocket-plugin/src/public_api.ts

Reply:
Add all

```
ConnectWebSocket,
  SendWebSocketMessage,
  DisconnectWebSocket,
  WebSocketDisconnected,
  WebsocketMessageError
```

Reply:
Because it was necessary to do it initially, they forgot to add it last time.

```ts
constructor(private router: Router, private actions$: Actions) {
    this.actions$
      .pipe(ofActionDispatched(DisconnectWebSocket))
      .subscribe(({ payload }) =>console.log('disconnect');
  }
```

Reply:
There is already this line in public_api, not sure what you want to achieve and how is it relevant to this PR, I suggest create issue to track existing backlog
export * from './symbols';